content/docs/0.14.x/reference/bridge/manage_sequences/index.md

```diff
@@ -5,6 +5,90 @@ weight: 40
 keywords: [0.14.x-bridge]
 ---
 
+## Trigger new sequences
+
+To trigger new sequences, head to the environment view. There you can find the button "Trigger a new sequence". This button can also be found on the sequences view, and will redirect you to the the environment view too.
+
+{{< popup_image
+link="./assets/trigger-01-environment-screen.png"
+caption="Sequences">}}
+
+{{< popup_image
+link="./assets/trigger-02-sequence-screen.png"
+caption="Sequences">}}
+
+The first step in triggering a sequence is to select the sequence, as well as the service and stage you want to trigger the sequence for. For sequences you have the possibility to choose from 3 sequence types:
+* Delivery - This will trigger the `delivery` sequence as defined in the [shipyard](https://keptn.sh/docs/0.14.x/manage/shipyard)
```

Comment:
I would not use the hard link so we don't depend on version (or domain) :) 
Please check my suggestion if there is the correct number of parent directories
```suggestion
* Delivery - This will trigger the `delivery` sequence as defined in the [shipyard](../../manage/shipyard)
```